pkgs/by-name/ka/kaitai-struct-cpp-stl-runtime/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  gtest,
+  zlib,
+  copyPkgconfigItems,
+  makePkgconfigItem,
+  lib,
+  testers,
+  ctestCheckHook,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "kaitai-struct-cpp-stl-runtime";
+  version = "0.11";
+
+  src = fetchFromGitHub {
+    owner = "kaitai-io";
+    repo = "kaitai_struct_cpp_stl_runtime";
+    tag = finalAttrs.version;
+    sha256 = "sha256-2glGPf08bkzvnkLpQIaG2qiy/yO+bZ14hjIaCKou2vU=";
+  };
+
+  doCheck = true;
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    copyPkgconfigItems
+    ctestCheckHook
+  ];
+
+  buildInputs = [
+    zlib.dev
+    gtest
+  ];
+
+  strictDeps = true;
+
+  pkgconfigItems = [
+    (makePkgconfigItem rec {
```

Comment:
Can you open up at least an issue upstream (PR would be ideal, but not a blocker) asking them to do pkg config too?

Can then link the issue/PR here.

Reply:
Done.